siliconcompiler/core.py

```diff
@@ -563,40 +567,84 @@ def load_target(self, name):
 
         Args:
             name (str): Module name
-            flow (str): Target flow to
+            **kwargs (str): Options to pass along to the target
 
         Examples:
-            >>> chip.load_target('freepdk45_demo')
-            Loads the 'freepdk45_demo' target
-
+            >>> chip.load_target('freepdk45_demo', syn_np=5)
+            Loads the 'freepdk45_demo' target with 5 parallel synthesis tasks
         """
 
         # Record target
         self.set('option', 'target', name)
 
         load_function = self.find_function(name, 'setup', 'targets')
-        load_function(self)
+        load_function(self, **kwargs)
 
     ##########################################################################
-    def use(self, module):
+    def use(self, module, **kwargs):
         '''
         Loads a SiliconCompiler module into the current chip object by calling
         a module.setup() method.
         '''
 
+        # Load supported types here to avoid cyclic import
+        from siliconcompiler import PDK
+        from siliconcompiler import Flow
+        from siliconcompiler import Library
+        from siliconcompiler import Checklist
```

Comment:
Seems funky, what exactly is causing this issue? (is it related to the imports in __init__.py that bring things into the module scope?)

Reply:
I think the custom `__init__.py` causes circular import issues if you try to use anything that it imports (or `import siliconcompiler`) within `core.py`.

You might be able to put something like this at the top of the file instead:

`from siliconcompiler.use import PDK, Flow, Library, Checklist`

Reply:
@WRansohoff is right. I can try to see if there is a good method to avoid this.